py/selenium/webdriver/common/actions/key_actions.py

```diff
@@ -28,19 +28,7 @@ class KeyActions(Interaction):
     def __init__(self, source: KeyInput | PointerInput | WheelInput | None = None) -> None:
         if source is None:
             source = KeyInput(KEY)
-        self.input_source = source
```

Comment:
Can you leave this line in? We don't use it anywhere else in codebase, but it's not a private attribute and could possibly break some user code that might be accessing it.